docs/content/engines/trino.md

```diff
@@ -26,6 +26,263 @@ under the License.
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [358, 368)    | [paimon-trino-358-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-358/{{< version >}}/paimon-trino-358-{{< version >}}.jar)    |
+| [368, 369)    | [paimon-trino-368-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-368/{{< version >}}/paimon-trino-368-{{< version >}}.jar)    |
+| [369, 370)    | [paimon-trino-369-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-369/{{< version >}}/paimon-trino-369-{{< version >}}.jar)    |
+| [370, 388)    | [paimon-trino-370-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-370/{{< version >}}/paimon-trino-370-{{< version >}}.jar)    |
+| [388, 393)    | [paimon-trino-388-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-388/{{< version >}}/paimon-trino-388-{{< version >}}.jar)    |
+| [393, latest] | [paimon-trino-393-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-trino-393/{{< version >}}/paimon-trino-393-{{< version >}}.jar)    |
+
+{{< /stable >}}
+
+{{< unstable >}}
+
+| Version       | Jar                                                                                                                                 |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| [358, 368)    | [paimon-trino-358-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-358/{{< version >}}/) |
+| [368, 369)    | [paimon-trino-368-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-368/{{< version >}}/) |
+| [369, 370)    | [paimon-trino-369-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-369/{{< version >}}/) |
+| [370, 388)    | [paimon-trino-370-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-370/{{< version >}}/) |
+| [388, 393)    | [paimon-trino-388-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-388/{{< version >}}/) |
+| [393, latest] | [paimon-trino-393-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-trino-393/{{< version >}}/) |
+
+{{< /unstable >}}
+
+You can also manually build bundled jar from the source code.But before compiling, you need to do a few things first:
+
+- To build from source code, [clone the git repository]({{< trino_github_repo >}}).
```

Comment:
```suggestion
- To build from the source code, [clone the git repository]({{< trino_github_repo >}}).
```

Reply:
Thanks for your review, I will revise this.